src/python/zquantum/core/interfaces/backend.py

```diff
@@ -41,6 +41,38 @@ def get_expectation_values(self, circuit:Circuit, qubit_operator:QubitOperator,
         """
         raise NotImplementedError
 
+    def get_expectation_values_for_circuitset(self, circuitset:[Circuit], qubit_operator:QubitOperator, **kwargs) -> [ExpectationValues]:
```

Comment:
Actually, according to https://docs.python.org/3/library/typing.html, it should be:
`circuits:List[Circuit]` (with `from typing import List` at the top)

Reply:
Addressed here:
68c0a60